clicky/runClickyScript.py

```diff
@@ -0,0 +1,10 @@
+from clickyScript import *
+
+
+clickyUrl = "https://api.clicky.com/api/stats/4?site_id=101369228&sitekey=6d6a506f44d45a59&type=visitors-list&date=last-week&output=json"
```

Comment:
I wonder if we should move the `site_id` and `sitekey` parameters to environment variables? We already have the `site_id` in the `.env` file.

- Our site details: https://clicky.com/stats/prefs?site_id=101369228
- Info on `site_id and `sitekey`: https://clicky.com/help/api

It looks like like `sitekey` is almost intended as a password to access the API.

Reply:
Awesome! I just tested locally and via the GitHub Action on my cloned repo, and both scenarios were still able to access the correct URL using the environment variables.